### ISSUE TITLE
Few missing material recipes in tailoring overhaul

### ISSUE THREAD

**Issue by a-chancey** [CleverRaven/Cataclysm-DDA#58504]
### Describe the bug

I done goofed and somehow the recipes for a few of the patchwork sheets got misplaced. Leather, fur, faux fur, synthetic fabric (for which the patchwork sheet is incorrectly named still, going to fix that as well), and i believe Lycra. 

Again, oops. When I have wifi tomorrow, I’ll write the PR. 

### Steps to reproduce

1. Try to make the above mentioned patchwork sheets. 
2. Notice that I somehow missed the recipes. 

### Expected behavior

I should be able to sew leather together into a patchwork sheet. That’s why I did all this damn work. 

### Versions and configuration

Not relevant, just any after #54256 was merged. 

### Additional context

Again I can fix it by submitting a new PR tomorrow or I can write it up in notes on my phone and send it to someone
